orchestra/interface_api/project_management/project_management.py

```diff
@@ -51,7 +52,7 @@ def project_management_information(project_id):
 
 def edit_slack_membership(project_id, username, action):
     slack = OrchestraSlackService()
-    slack_user_id = slack.users.get_user_id(username)
+    slack_user_id = Worker.objects.get(user__username=username).slack_user_id
```

Comment:
yeah here's where we made projman saner: use the worker username, not the slack username.